src/main/java/com/google/sps/servlets/AuthorizationServlet.java

```diff
@@ -40,93 +40,30 @@
     UNKNOWN;
   }
 
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("Entered doPOST method");
+
+    // Creating a user from the registration page
+    final String emailEntered = request.getUserPrincipal().toString();
+    final String typeEntered = processType(request).toString();
+
+    // Creating an UserEntity
+    Entity taskEntity = new Entity("User Data");
```

Comment:
I think we should avoid using spaces in our entity names and keys, so User Data -> UserData